appcues/src/main/java/com/appcues/debugger/AppcuesDebuggerManager.kt

```diff
@@ -72,16 +68,15 @@ internal class AppcuesDebuggerManager(
         coroutineScope.launch {
             viewModel.uiState.collect { state -> onBackPressCallback.isEnabled = state is Expanded }
         }
-        addDebuggerView(activity, viewModel)
-        application.registerActivityLifecycleCallbacks(this)
-        setDebuggerBackPressCallback(activity)
+        addDebuggerView(viewModel)
+        contextWrapper.getApplication().registerActivityLifecycleCallbacks(this)
```

Comment:
noting this moved back to not use the ActivityMonitor singleton, so that should work well with the tests now.